cypress/integration/ansiiLogs/ansii_logView.ts

```diff
@@ -31,41 +34,44 @@ describe("Basic evergreen log view", () => {
     cy.dataCy("log-row-2").should("be.visible");
     cy.dataCy("log-row-2").should(
       "contain.text",
-      "disableLogicalSessionCacheRefresh"
+      `[2022/03/02 17:01:58.701] Running pre-task commands.`
     );
     cy.dataCy("log-row-2").isContainedInViewport();
```

Comment:
I actually think you want these to all be `log-row-22` rather than `log-row-2`, based on the corresponding resmoke test. (i.e. in the previous test we check that `log-row-22` is not captured in the viewport due to wrap being disabled, in this test we want to test that `log-row-22` is captured in the viewport due to wrap being enabled)

Reply:
oops great catch!